modules/k3s/k3s_image_test.go

```diff
@@ -101,8 +101,8 @@ func Test_LoadImages(t *testing.T) {
 		if err != nil {
 			t.Fatal(err)
 		}
-
-		if pod.Status.ContainerStatuses[0].State.Waiting.Reason == "ErrImageNeverPull" {
+		waiting := pod.Status.ContainerStatuses[0].State.Waiting
+		if waiting != nil && waiting.Reason == "ErrImageNeverPull" {
```

Comment:
Could `waiting` be nil? If so, how we are handling it in tests?

Reply:
If `waiting is `nil` it means the pod is not waiting for the image, so it was properly loaded. We are not interested any other case.